plugins/p2p/p2p.go

```diff
@@ -21,25 +21,25 @@ func createManager(lPeer *peer.Local) *p2p.Manager {
 	// resolve the bind address
 	localAddr, err = net.ResolveTCPAddr("tcp", Parameters.BindAddress)
 	if err != nil {
-		Plugin.LogFatalfAndExit("bind address '%s' is invalid: %s", Parameters.BindAddress, err)
+		Plugin.LogFatalfAndExitf("bind address '%s' is invalid: %s", Parameters.BindAddress, err)
```

Comment:
I think this should not be marked as change in our PR, as we merged this from develop, no?